.github/workflows/on-change.yaml

```diff
@@ -24,7 +24,7 @@ jobs:
       - name: Run Gosec Security Scanner
         uses: securego/gosec@master
         with:
-          args: ./...
+          args: -exlude G104 ./...
```

Comment:
Why do we exactly exclude this rule?

Reply:
I would say it is a pretty important rule too: https://securego.io/docs/rules/g104.html

Reply:
This is making the workflow fail with some errors that do not make much
sense.
I am looking to just ditch it altogether.

Reply:
Keeping it for now, fixed the failing lines.